be/src/runtime/fragment_mgr.cpp

```diff
@@ -510,7 +510,10 @@ void FragmentMgr::_exec_actual(std::shared_ptr<FragmentExecState> exec_state, Fi
         std::lock_guard<std::mutex> lock(_lock);
         _fragment_map.erase(exec_state->fragment_instance_id());
         if (all_done && fragments_ctx) {
-            _fragments_ctx_map.erase(fragments_ctx->query_id);
+            auto search = _fragments_ctx_map.find(fragments_ctx->query_id);
```

Comment:
why search it first? I think map.erase method will do this automanticly.

Reply:
Indeed, fix